packages/app/background-jobs-common/src/background-job-processor/processors/AbstractBackgroundJobProcessor.ts

```diff
@@ -429,24 +409,6 @@ export abstract class AbstractBackgroundJobProcessor<
     })
   }
 
-  logJobStarted(job: JobType, requestContext: RequestContext): void {
-    requestContext.logger.info(
-      {
-        origin: this.constructor.name,
-      },
-      `Started job ${job.name}`,
-    )
-  }
-
-  logJobFinished(job: JobType, requestContext: RequestContext, isSuccess: boolean): void {
-    requestContext.logger.info(
-      {
-        isSuccess,
-      },
-      `Finished job ${job.name}`,
-    )
-  }
-
```

Comment:
Why do we need this to be public, even protected sounds weird to me, class should have very basic logging for start and finish, if most complex logging is required we have hooks to allow to implement what is needed

Reply:
Main reason is to be able to configure what fields and in what structure get logged, so e. g. you could add dumping of entire payload under debug level. Having it as `protected` is sufficient for that purpose, though.

You are right that hooks should be a valid replacement for that, but having an example (or a reference) somewhere in documentation about them existing and how to use them would be helpful if that is our preferred way.

Reply:
Yeah, the doc about hooks is missing 😓, if you agree with this change we should improve the overall doc, and in this particular case I think  we can create a `UPGRADING.md` to explain breaking changes and how to approach them

Reply:
yeah, that sounds great!